model/api_operations.js

```diff
@@ -66,6 +66,58 @@ function get_user_info(name)
     });
 }
 
+function submit_video(video_id, which_channel, seconds, who_submit) {
+    // create a new entry in the table named "video"
+    // also create a new entry in the table named "voting"
+    // using findOrCreate https://sequelize.readthedocs.io/en/2.0/docs/models-usage/
+    /*User
+        .findOrCreate({ where: { username: 'sdepold' }, defaults: { job: 'Technical Lead JavaScript' } })
+        .spread((user, created) => {
+            console.log(user.get({
+                plain: true
+            }))
+            console.log(created)
+
+            /*
+             findOrCreate returns an array containing the object that was found or created and a boolean that will be true if a new object was created and false if not, like so:
+        
+            [ {
+                username: 'sdepold',
+                job: 'Technical Lead JavaScript',
+                id: 1,
+                createdAt: Fri Mar 22 2013 21: 28: 34 GMT + 0100(CET),
+                updatedAt: Fri Mar 22 2013 21: 28: 34 GMT + 0100(CET)
+              },
+              true ]
+        
+         In the example above, the "spread" on line 75 divides the array into its 2 parts and passes them as arguments to the callback function defined beginning at line 39, which treats them as "user" and "created" in this case. (So "user" will be the object from index 0 of the returned array and "created" will equal "true".)
+            *//*
+        }) */
```

Comment:
Would prefer removal of unnecessary code comments. It would be nice to have stylistic consistency too with rest of the code like braces, spacing, max line width less than 82 characters etc - if you have extra time.

Reply:
yea sure